packages/coinjoin/src/backend/CoinjoinBackendClient.ts

```diff
@@ -130,41 +128,29 @@ export class CoinjoinBackendClient {
         }
     }
 
-    private fetchFromBlockbook<T extends keyof BlockbookWS>(
-        options: RequestOptions | undefined,
-        method: T,
-        ...params: Parameters<BlockbookWS[T]>
-    ) {
+    private fetchFromBlockbook<T>(
+        handler: (api: BlockbookAPI) => T | Promise<T>,
```

Comment:
```suggestion
        requestFn: (api: BlockbookAPI) => T | Promise<T>,)
```
nitpick: this isn't really a handler, right? more like a callback done after successful connection so:
requestFn, callbackFn, apiMethodFn ...

Reply:
Agreed, I've changed also wasabi handler to `callbackFn` (for sake of consistency) and renamed `fetchFromBlockbook` to `getBlockbookApi` (as it doesn't necessarily fetch anything now).

https://github.com/trezor/trezor-suite/pull/9101/commits/ea7fb2401ed8d7013d26da4fb17e34471f74ffb5 https://github.com/trezor/trezor-suite/pull/9101/commits/75d64d8702d12507377544307835150accf02286 https://github.com/trezor/trezor-suite/pull/9101/commits/07a6adb205c68449f34bfc8de83d06021a3b5736